src/components/Footer.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import logo from '../images/logo.svg';
 
 import {
   Grid,
   Header,
   Icon,
   Segment,
+  Image
```

Comment:
Note for us for followup (no worries to @misspran, we can take care of this one if you want): This, along with a couple other small things, are causing lint warnings. Someone should run `npm run lint` when we get a chance.